### ISSUE TITLE
keycloak user is already defined

### ISSUE THREAD

**Issue by UbaidurRehman1** [UbaidurRehman1/Cloud-Native-App-Spring-Boot#282]
`envcn/docker-compose.yml`
```
        environment:
            KEYCLOAK_USER: 'ubaid'
#            KEYCLOAK_USER: 'ubaid'
```

If auth-database is already populated, then we get keycloak user is already define when we re-run the application.
Quick Fix:
Please comment out `KEYCLOAK_USER: 'ubaid' ` in encvn/docker-compose.yml file.

**Comment by UbaidurRehman1** [comment]
Reproduce this issue

**Comment by UbaidurRehman1** [comment]
- [ ] Reproduce this issue
- [ ] Fix

**Comment by UbaidurRehman1** [comment]
Seems like this bug has been fixed in KeyCloak v12.0.1 as I could not reproduce this issue in my local environment.
